tree/dataframe/inc/ROOT/RDF/InterfaceUtils.hxx

```diff
@@ -663,7 +662,7 @@ struct RNeedJittingHelper<RInferredType> {
    static constexpr bool value = true;
 };
 
-template <typename ...ColTypes>
+template <typename... ColTypes>
```

Comment:
there is a number of unrelated whitespace changes in the commit, can you please remove them?

(`clang-format FILENAME` will reformat the whole file, `git clang-format --diff master HEAD | git apply` will reformat only lines around those the commits in range `master..HEAD` touched)

Reply:
Okay noted - I should have asked which would have been better, sorted now!

Reply:
Excess whitespace is no more :)

Reply:
I would have been ok with the unrelated whitespace changes in a separate commit -- but it's also ok to leave them as they are